docs/simulations.rst

```diff
@@ -32,6 +32,14 @@ dividing by the average mass (mass prior model).  This number of stars is
 simulated at each age and then the completeness function is applied to remove
 all the simulated stars that would not be observed.
 
+The `beast simulate_obs` also can be used to just simulate extra ASTs. This is very
+powerful if you need more ASTs for generating MATCH-optimized ASTs in addition to what 
+you already have from the DOLPHOT ASTs. This means that you can avoid extensive/additional
+DOLPHOT ASTs as long as your initial/existing DOLPHOT ASTs properly cover the entire
+CMD space (i.e., meeting the minimum requirement by MATCH should be enough.) by
+providing the information on the flux uncertainties, biases, and the completeness
+for the additionally selected model SEDs.
```

Comment:
I feel a bit more detail and explanation would be good here.  Maybe state that the simulation is *replacing* generating additional input ASTs and running those through DOLPHOT.  Might be useful to state that the initial/existing ASTs generated to be optimized for the toothpick model (BEAST-optimized?), provide the needed coverage.

Also, I think it would be better to be general about the photometry routine as there is not specific dependence on ASTs and the BEAST for DOLPHOT (even if that is really only what we've used).

Reply:
I tried to reflect your comments as much as possible. Based on my tests so far, we need BEAST-optimized (toothpick) + MATCH-friendly(random in given magnitude and color ranges) as the existing ASTs from which we construct a noisemodel grid. But the real ASTs do not need to densely cover the CMD; uniform and complete coverage is more important. We can update this documentation later when we have more robust test results.